metacat-connector-polaris/src/main/java/com/netflix/metacat/connector/polaris/PolarisConnectorDatabaseService.java

```diff
@@ -114,6 +114,13 @@ public void update(final ConnectorRequestContext context, final DatabaseInfo dat
                 params.putAll(databaseInfo.getMetadata());
             }
             db.setParams(params);
+            // Temporarily setting location/audit from databaseInfo for backfill
+            if (databaseInfo.getUri() != null) {
+                db.setLocation(databaseInfo.getUri());
+            }
+            if (databaseInfo.getAudit() != null && databaseInfo.getAudit().getCreatedBy() != null) {
```

Comment:
similar q as above. modifications to createdBy dont seem as risky, but im wondering if this field is inherited from db to table anywhere that could have unintended consequences